web/styles/subscriptions.css

```diff
@@ -117,7 +115,14 @@ h3.user_group_setting_subsection_title {
     line-height: 1.5;
 }
 
-h4.stream_setting_subsection_title,
+h4.stream_setting_subsection_title {
+    display: inline-block;
+    font-size: 1.35em;
+    font-weight: normal;
+    line-height: 1.5;
+    margin-bottom: 5px;
+}
```

Comment:
There is no need to duplicate all this CSS. You can just add a margin rule for `h4.stream_setting_subsection_title` separately.

Reply:
I've now added the margin rule separately as suggested.

Reply:
I've also made the improvements to the commit message by updating the line-wrapping and added a period after the "Fixes..." line.